bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/widgets/Display.java

```diff
@@ -6946,4 +6946,7 @@ static long windowProc(long id, long sel, long arg0, long arg1, long arg2, long
 	}
 }
 
+static boolean isActivateShellOnForceFocus() {
+	return "true".equals(System.getProperty("org.eclipse.swt.internal.activateShellOnForceFocus", "false")); //$NON-NLS-1$
```

Comment:
`Boolean.getBoolean()` ?

Reply:
No, because with `Boolean.getBoolean()` you don't have control over the default if no option is set.

Reply:
I guess this depends on the expectation of how likely is it that default will be changed. If not too likely, then simpler code is better. Afterall the one who'll change the default could rewrite the code back to how it is now. And if complexity can be done later at the same cost, then there's no need to add it now. Anyways, if you believe that it's likely to be changed, it's fine to keep is as is.